sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -227,9 +228,10 @@ struct _pi_context : _pi_object {
 
   // Mutex Lock for the Command List Cache
   std::mutex ZeCommandListCacheMutex;
```

Comment:
update the comment to say that it serves both caches of command-lists, but we could split this into two if needed

Reply:
Please don't split it.  I was the one that requested it only be one.  Ideally (but certainly not for thsi changeset) the whole command list caching scheme itself would be hidden behind a programmatic interface, and that you'd just request a command list of the proper type, and not be worried how the caches were implemented under the covers.

Reply:
I am fine with having just one as I don't believe it could be a perf. bottleneck.
But please indicate this in the comment

Reply:
Added comment

Reply:
Added a comment. Thanks